Conversion-Class Temperatura

```diff
@@ -0,0 +1,18 @@
+package com.company;
+
+public class Temperatura {
+
+    public double temperatura;
+    public void setTemperatura(double temperaturaZilei) {
+        this.temperatura = temperaturaZilei;
+    }
+    public double getTemperatura(){
+        return temperatura;
+    }
+
+    public void temperatura (double c, double f) {
+        double celsius = 0;
+        double fahrenheit = 70;
```

Comment:
`celsius` and `fahrenheit` are not used anywhere in this function, do you think you can remove them, or you need to use them somewhere else?